src/index.ts

```diff
@@ -876,7 +876,22 @@ const getAutoOpenRoom = (): string | undefined => {
   }
 };
 
+const notifyUID = "notifyUID";
```

Comment:
Should this not actually be a unique identifier?  Or is this just ensuring that we only show one message from brave talk at a time - in which case would it not make more sense for the constant value to be "brave-talk-notice" or similar?

Reply:
> Should this not actually be a unique identifier? Or is this just ensuring that we only show one message from brave talk at a time - in which case would it not make more sense for the constant value to be "brave-talk-notice" or similar?

I like your suggestion. It just needs to be a string that will not be reused, and something that is meaningful, and unlikely to be used by the gypsy internals, is a better choice.